supported_formats_and_operations.md

```diff
@@ -23,95 +23,96 @@ Which consumer software supports which format?
 | -------------------------- | ------- | ---------- | ---- |
+|  `binarize`                | no      | ?          | ?    |
 |  `clip`                    | ?       | ?          | ?    | 
-|  `min_max`                 | ?       | ?          | ?    |
-|  `percentile`              | ?       | ?          | ?    |
-| `scale_linear`             | ?       | ?          | ?    |
+|  `scale_linear`            | ?       | ?          | ?    |
+|  `sigmoid`                 | yes     | ?          | ?    |
 |  `zero_mean_unit_variance` | yes     | ?          | ?    |
 
 
-# Postprocessing
+## Preprocessing
 
-The supported postprocessing operations.
+Additional preprocessing operations.
 
-- `binarize` binarize the tensor with a fixed threshold, values above the threshold will be set to one, values below the threshold to zero
+- `scale_range` normalize the tensor with percentile normalization
   - `kwargs`
-    - `threshold` the fixed threshold
-  - `reference_implemation`
-- `clip` clip the tensor, same as in [preprocessing](https://github.com/bioimage-io/configuration/blob/master/supported_formats_and_operations.md#preprocessing).
-- `sigmoid` apply a sigmoid to the tensor.
-  - `kwargs` None
-  - `reference_implementation`
-- `scale_mean_variance` scale the tensor s.t. its mean and variance match a reference input 
+    - `mode` can be one of `per_sample` (percentiles are computed for each sample individually), `per_dataset` (percentiles are computed for the entire dataset). For a fixed scaling use `scale linear`.
+    - `axes` the subset of axes to normalize jointly. For example `xy` to normalize the two image axes for 2d data jointly. The batch axis (`b`) is not valid here.
+    - `min_percentile` the lower percentile used for normalization, in range 0 to 100. Default value: 0.
+    - `max_percentile` the upper percentile used for normalization, in range 0 to 100. Has to be bigger than `upper_percentile`. Default value: 100.
+  - `reference_implementaion`
+
+
+### Consumers
+
+Which consumer supports which preprocessing operation?
+
+| preprocesing               | ilastik | deepImageJ | Fiji |
+| -------------------------- | ------- | ---------- | ---- |
+|  `scale_range`             | ?       | ?          | ?    |
+
+
+## Postprocessing
+
+Additional postprocessing operations.
+
+- `scale_range` normalize the tensor with percentile normalization
   - `kwargs`
-    - `mode` one of `per_dataset` or `per_sample` (for fixed mean and variance use `scale_linear`)
-    - `reference_input` name of the input tensor to use as reference
+    - same as preprocessing
+    - `reference_tensor` tensor name to compute the percentiles from. Default: The tensor itself. If `mode`==`per_dataset` this needs to be the name of an input tensor.
   - `reference_implementation`
-- `scale_min_max` scale the tensor s.t. its min and max match a reference tensor
+- `scale_mean_variance` scale the tensor s.t. its mean and variance match a reference tensor 
```

Comment:
Also refere to the preprocessing here.

Reply:
should `scale_mean_variance` exist for preprocessing? It is different from the `zero_mean_unit_variance`.